Modules/System/Web Service Management/src/WebServiceManagement.Codeunit.al

```diff
@@ -238,4 +238,13 @@ codeunit 9750 "Web Service Management"
     begin
         exit(WebServiceManagementImpl.IsServiceNameValid(Value));
     end;
+
+    /// <summary> 
+    /// Deletes a webservice.
+    /// </summary>
+    /// <param name="WebServiceAggregate">The record to be deleted.</param>
+    procedure DeleteWebService(var WebServiceAggregate: Record "Web Service Aggregate")
```

Comment:
Currently, there's an OnDelete trigger on `Web Service Aggregate` so this is just to avoid wrting `WebServiceAggregate.Delete(true)`?

```
    trigger OnDelete()
    var
        WebServiceManagementImpl: Codeunit "Web Service Management Impl.";
    begin
        WebServiceManagementImpl.DeleteWebService(Rec);
    end;
```

Reply:
Ok i didn´t know about this, but if we try to use it as such we still get this: error AS0059: The table 'Web Service Aggregate' cannot be modified because it is part of the read-only application database or because it will result in undefined behavior.

Reply:
Oh, I should have read the issue, thanks for pointing it out.

In this case, just one minor thing: do you think it is a good idea to provide some defensive checks? Like, what if someone istrying to delete a published web service?

Reply:
I am not sure maybe it might be better to implement it like you do with createwebservice and createtenantwebservice where createwebservice is onprem.
would be something like this?

// Web Service Management:
[Scope('OnPrem']
procedure RemoveWebService(var WebServiceAggregate: Record "Web Service Aggregate")
begin
    WebServiceManagementImpl.DeleteWebService(WebServiceAggregate);
end;


procedure RemoveTenantWebService(var WebServiceAggregate: Record "Web Service Aggregate")
begin
    WebServiceManagementImpl.DeleteTenantWebService(WebServiceAggregate);
end;



// Web Service Management Impl.:
procedure RemoveWebService(var Rec: Record "Web Service Aggregate")
var
WebService: Record "Web Service";
begin
    if WebService.Get(Rec."Object Type", Rec."Service Name") then
        WebService.Delete();
end;


procedure RemoveTenantWebService(var Rec: Record "Web Service Aggregate")
var
TenantWebService: Record "Tenant Web Service";
begin
    if TenantWebService.Get(Rec."Object Type", Rec."Service Name") then
        TenantWebService.Delete();
end;